MTM/__init__.py

```diff
@@ -89,6 +94,13 @@ def findMatches(image,
                       xOffset:xOffset+searchWidth]
     else:
         xOffset = yOffset = 0
+    
+    # TODO dont change listTemplate in-place + use a list comprehension
+    if downscaling_factor != 1:
+        image = transform.rescale(image, 1/downscaling_factor, anti_aliasing = False)
+        
+        for i in range(len(listTemplates)):
+            listTemplates[i] = transform.rescale(listTemplates[i], 1/downscaling_factor, anti_aliasing = False)
```

Comment:
attention ici on modifie/overwrite la liste listTemplate donne en input, ca peut poser probleme si on veut conserver la liste originale inchangee.
A la place on peut re-creer une liste avec une list-comprehension
`listTemplate = [transform.rescale(template, 1/downscaling_factor, anti_aliasing = False) for template in listTemplate]`

Ca ne devrait pas change la liste donne en parametres de la fonction, comme on declare la variable listTemplate localement au sein de la fonction